app/frontend/components/pages/sendAda/sendAdaPage.tsx

```diff
@@ -330,7 +330,7 @@ const SendAdaPage = ({
       <label className="send-label">Asset</label>
       <SearchableSelect
         wrapperClassName="no-margin"
-        defaultItem={selectedAsset}
+        selectedItem={selectedAsset}
```

Comment:
as far as I checked, `value` seems to be the prevalent name to refer to the current form element value, e.g. in input fields, so I'd consider naming this "value" for the sake of consistency of the general interface of form inputs. I don't see a discrepancy with `displaySelectedItem` nor ` displaySelectedItemClassName` as these refer to properties specific to the select component which indeed has a notion of "selected item"